drawings.h

```diff
@@ -4,11 +4,123 @@
 extern "C" {
 #endif
 
+  typedef struct GOffscreen GOffscreen;
```

Comment:
I'm curious, how does it work. Do we define here GOffscreen as an undefined type that will be used only as a pointer?
Wouldn't it be clearer to define it as uintptr_t - similarily to how fltk itself is doing it?